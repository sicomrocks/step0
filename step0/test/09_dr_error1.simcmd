#test des paramÃ¨tres de la fonction dr ; registre incorrect

dr HI
dr truc #ce registre n'existe pas
dr LO
