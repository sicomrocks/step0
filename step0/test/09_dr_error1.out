HI : 0x00000000
